src/commands/createStaticWebApp/DeploymentMethodStep.ts

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, IAzureQuickPickItem, IWizardOptions } from "vscode-azureextensionui";
+import { ext } from "../../extensionVariables";
+import { localize } from "../../utils/localize";
+import { openUrl } from "../../utils/openUrl";
+import { RepoCreateStep } from "../createRepo/RepoCreateStep";
+import { WorkspaceListStep } from "../createRepo/WorkspaceListStep";
+import { GitHubBranchListStep } from "./GitHubBranchListStep";
+import { GitHubOrgListStep } from "./GitHubOrgListStep";
+import { GitHubRepoListStep } from "./GitHubRepoListStep";
+import { IStaticWebAppWizardContext } from "./IStaticWebAppWizardContext";
+
+export type DeploymentMethods = 'local' | 'existingRepo' | 'learnMore';
```

Comment:
The 'learnMore' option is only used in this file, so I would not include it in the type exported to other files. I'd probably do `IAzureQuickPickItem<DeploymentMethods | undefined>` and make data `undefined` for the learnMore pick